taf/testlib/virtual_env.py

```diff
@@ -28,7 +28,9 @@
 #  "instance_type": "settings",
 #  "id": "993",
 #  "images_share_path": "/mnt/berta/oses/openstack",
-#  "mgmt_ip_cidr": "11.212.23.11/8",
+#  "external_net_ip_cidr": "11.212.23.0/24",
+#  "external_net_pool_start": "100",
+#  "external_net_pool_end": "199",
```

Comment:
11.0.0.0/8 is owned by the DoD.  Do not use it as an example.

11.0.0.0/8	DoD Intel Information Systems	IANA Date 1993-05